oops/oops_test.go

```diff
@@ -476,10 +488,13 @@ testing.tRunner
 		},
 	}
 
-	re := regexp.MustCompile(`\.(go|s):\d+`)
 	for _, testcase := range testcases {
 		t.Run(testcase.Title, func(t *testing.T) {
-			actualVerbose := re.ReplaceAllString(fmt.Sprint(testcase.Error), `.$1:123`)
+			actualVerbose := fixLineNumbers(fmt.Sprint(testcase.Error))
+			actualVerbose, err := stripPathPrefix(actualVerbose)
+			if err != nil {
+				t.Errorf(err.Error())
```

Comment:
Nonblocking nit: I see other tests in `oops_test.go` use `assert`. How do you feel about doing so here (and in the other test assertions added by this PR) as well, for consistency's sake? Or perhaps you feel there's a reason to deviate from that pattern?

Reply:
Those potential errors aren't really asserting against the functionality of the library per-se but yeah i don't see any reason not to unify it